dotcms-integration/src/test/java/com/dotcms/datagen/DotAssetDataGen.java

```diff
@@ -0,0 +1,52 @@
+package com.dotcms.datagen;
+
+import com.dotcms.contenttype.model.type.DotAssetContentType;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.folders.model.Folder;
+import java.io.File;
+import joptsimple.internal.Strings;
```

Comment:
internal class 
why not String.join(",", tags)

Reply:
oh sorry, I see, let me fix

Reply:
good eye Fabro!  The imports were folded in my IDE and I did not see which `Strings` I was using.